rustport/cppshim/include/bindings.hpp

```diff
@@ -0,0 +1,40 @@
+#include "bindings.h"
+
+#include <iostream>
+#include <type_traits>
+#include <utility>
+
+namespace rust {
+
+/// @brief Asynchronously invoke f on the rust-backed executor.
```

Comment:
Our documentation system is Hyde based - Although Hyde 2.0 has some support for doxygen style comments I'm not sure how this will play. The Hyde 2.0 work is currently in my fork https://github.com/sean-parent/stlab-libraries.